packages/client/src/components/Question/QuestionAnswer.tsx

```diff
@@ -1,13 +1,14 @@
 import { Paragraph } from "@amsterdam/asc-ui";
-import { Answer, ClientOutcomes } from "@vergunningcheck/imtr-client";
+import type { AnswerType } from "@vergunningcheck/imtr-client";
+import { ClientOutcomes } from "@vergunningcheck/imtr-client";
```

Comment:
It's a default export so please import from the root.

Reply:
Not sure what you mean. It is imported from the root.
Please note line 2 is for importing `types` only. Line 3 is for importing actual variables.

Reply:
Ah okay, sorry, I thought it was the same as the previous comment. 

You can keep it like this, in my PR I am converting this to:
```
import * as imtr from "@vergunningcheck/imtr-client";
```

Reply:
Okay